src/DevBetterWeb.UploaderApp/Builders.cs

```diff
@@ -30,6 +30,21 @@ public static GetAllVideosService BuildGetAllVideosService(HttpService httpServi
       return new GetAllVideosService(httpService, _loggerFactory.CreateLogger<GetAllVideosService>());
     }
 
+    public static GetAnimatedThumbnailService BuildGetAnimatedThumbnailService(HttpService httpService)
+    {
+      return new GetAnimatedThumbnailService(httpService, _loggerFactory.CreateLogger<GetAnimatedThumbnailService>());
+    }
+
+    public static GetStatusAnimatedThumbnailService BuildGetStatusAnimatedThumbnailService(HttpService httpService)
+    {
+      return new GetStatusAnimatedThumbnailService(httpService, _loggerFactory.CreateLogger<GetStatusAnimatedThumbnailService>());
+    }
+
+    public static AddAnimatedThumbnailsToVideoService BuildAddAnimatedThumbnailsToVideoService(HttpService httpService)
+    {
+      return new AddAnimatedThumbnailsToVideoService(httpService, _loggerFactory.CreateLogger<AddAnimatedThumbnailsToVideoService>());
+    }
+
```

Comment:
Why do we need these? Should we be able to just request them via constructor wherever needed, using DI?

Reply:
done on next PR